tests/test_tickets.py

```diff
@@ -43,8 +50,15 @@ def make_basic_fare_setup():
     assert Fare.objects.all().count() == 0
     conference_str = settings.CONFERENCE_CONFERENCE
 
-    Conference.objects.get_or_create(code=conference_str,
-                                     name=conference_str)
+    Conference.objects.get_or_create(
+        code=conference_str,
+        name=conference_str,
+        # using 2018 dates
+        # those dates are required for Tickets to work.
+        # (for setting up/rendering attendance days)
+        conference_start=date(2018, 7, 23),
```

Comment:
will be useless after ep2018...

Reply:
we could probably change that to date.today() but the main point here is that it needs *any* date to work – we don't care what the date is, but otherwise it would fail with Traceback on the profile view making the whole profile unreachable in tests.